test/e2e/atlas/streams_test.go

```diff
@@ -0,0 +1,317 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//go:build e2e || (atlas && generic)
+
+package atlas_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/mongodb/mongodb-atlas-cli/test/e2e"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	atlasv2 "go.mongodb.org/atlas-sdk/v20230201008/admin"
+)
+
+func TestStreams(t *testing.T) {
+	if IsGov() {
+		t.Skip("Skipping Streams integration test, Streams processing is not enabled in cloudgov")
+	}
+
+	g := newAtlasE2ETestGenerator(t)
+	g.generateProject("atlasStreams")
+
+	a := assert.New(t)
+	req := require.New(t)
+
+	cliPath, err := e2e.AtlasCLIBin()
+	req.NoError(err)
+
+	instanceName, err := RandEntityWithRevision("instance")
+	req.NoError(err)
+
+	connectionName, err := RandEntityWithRevision("connection")
+	req.NoError(err)
+
+	t.Run("List all streams in the e2e project", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			"streams",
+			"instance",
+			"list",
+			"-o=json",
+			"--projectId",
+			g.projectID,
+		)
+
+		cmd.Env = os.Environ()
+		resp, err := cmd.CombinedOutput()
+		req.NoError(err)
+
+		var instances atlasv2.PaginatedApiStreamsTenant
+		err = json.Unmarshal(resp, &instances)
+		a.NoError(err)
```

Comment:
empty checks wont' work if this fails
```suggestion
		require.NoError(t, json.Unmarshal(resp, &instances))
```

Reply:
Resolved in [d361753](https://github.com/mongodb/mongodb-atlas-cli/pull/2138/commits/d36175356da5aeaa288dee5357da95e9a8ffae4e)